src/Optimize.jl

```diff
@@ -16,6 +17,6 @@ include("solver/solver.jl")
 # Utilities.
 include("bmark/run_solver.jl")
 include("bmark/bmark_solvers.jl")
-include("bmark/jump_vs_ampl.jl")
+include("bmark/mpb_vs_ampl.jl")
```

Comment:
If we're not requiring JuMP, then this should only be `include`d if MathProgBase is available.